### PR TITLE
Add simpler methods for fetching blob paths to explorer

### PR DESCRIPTION
**Pull request by daniel-sol**

### CODE REVIEW

File: tests/context.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+def add_path():
+    """Way to add package path to sys.path for testing"""
+    # Adapted from https://docs.python-guide.org/writing/structure/
+    # Turned into function because the details here didn't work
+    package_path = str(Path.cwd() / "../src/")
+    while package_path in sys.path:
+        sys.path.remove(package_path)
+    sys.path.insert(0, package_path)
+
+add_path()
```
Comment (emirgens) [inline tests/context.py:15]:
```suggestion
add_path()
# add a comment
```

**Review by emirgens — CHANGES_REQUESTED** [review]
See my comment

**Review by emirgens — APPROVED** [review]

File: src/fmu/sumo/explorer/_case.py [hunk 2]
```diff
@@ -14,14 +18,38 @@ def __init__(self, sumo_client, meta_data):
...
         self.object_type = "case"
 
 
+    @property
+    def sumo_id(self):
+        """Returns attribute _sumo_id"""
+        return self._sumo_id
+
+    @sumo_id.setter
+    def sumo_id(self, sumo_id):
```
Comment (rwiker) [inline src/fmu/sumo/explorer/_case.py:34]: You probably should not have a setter for `_id`.

File: src/fmu/sumo/explorer/_case.py [hunk 3]
```diff
@@ -14,14 +18,38 @@ def __init__(self, sumo_client, meta_data):
...
+        """
+        self._sumo_id = sumo_id
+
+    @property
+    def name(self):
+        """returns name attribute"""
+        return self._name
+
+    @name.setter
+    def name(self, name):
```
Comment (rwiker) [inline src/fmu/sumo/explorer/_case.py:47]: Probably should not have a setter for `name`, either :-)

File: src/fmu/sumo/explorer/_explorer.py [hunk 4]
```diff
@@ -55,8 +70,34 @@ def get_status(self):
...
+            case_dict[result["_source"]["fmu"]["case"]["name"]] = result["_id"]
+        return case_dict
+
+    def get_case_by_name(self, name):
+        """Fetches case from case name
+        args:
+            name (str): name of case
+
+        """
+        query = f"class:case AND fmu.case.name:{name}"
```
Comment (rwiker) [inline src/fmu/sumo/explorer/_explorer.py:89]: This should probably be `f"class:case AND fmu.case.name:\"{name}\""`, if `name` can contain spaces, "/", ":" or other semi-exotic characters.

File: src/fmu/sumo/explorer/_utils.py [hunk 5]
```diff
@@ -14,21 +29,197 @@ class Property(str, Enum):
...
+        source = result["_source"]
+        if tagname is not None:
+            if source["data"]["tagname"] != tagname:
+                continue
+        if standard_get_name:
+            name = source["fmu"]["realization"]["id"]
+        else:
+            name = get_vector_name(source)
+
+        blobs[name] = source["_sumo"]["blob_url"]
```
Comment (rwiker) [inline src/fmu/sumo/explorer/_utils.py:209]: This may not be useful, as the `blob_url` does not contain an authorization token.

Comment (daniel-sol) [reply src/fmu/sumo/explorer/_utils.py]: Fixed: Now the function is renamed to get_object_blob_ids, and corresponding methods on class Case are renamed to get_blob_ids, and get_summary_blob_ids. All are tested with unit tests, and towards a webviz plugin.

File: src/fmu/sumo/explorer/_case.py (hunk 2 above)
Comment (daniel-sol) [reply src/fmu/sumo/explorer/_case.py]: Agree, fixed in commit 61bca33 on fork

File: src/fmu/sumo/explorer/_case.py (hunk 3 above)
Comment (daniel-sol) [reply src/fmu/sumo/explorer/_case.py]: Agree, fixed in commit 61bca33 on fork

File: src/fmu/sumo/explorer/_utils.py (hunk 5 above)
Comment (daniel-sol) [reply src/fmu/sumo/explorer/_utils.py]: And line changed to: blob_ids[name] = result["_id"]

**Review by rwiker — APPROVED** [review]
LGTM.

**Pull request merged**